src/Repository/OpdsAccessRepository.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\OpdsAccess;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+
+/**
+ * @extends ServiceEntityRepository<OpdsAccess>
+ */
+class OpdsAccessRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, OpdsAccess::class);
+    }
+
+    //    /**
```

Comment:
remove comments

Reply:
I would add a function `getByAccessToken` and already join the user in the query.